package.json

```diff
@@ -128,7 +128,7 @@
     "lodash.xor": "^4.5.0",
     "marked": "^0.3.5",
     "material-ui": "^0.18.7",
-    "material-ui-next": "1.0.0-beta.39",
+    "material-ui-next": "^1.0.0-beta.39",
```

Comment:
This was a workaround for the issue addressed in this PR. The Tooltips were hidden under the popup. Now that I'm using the v1 `Menu` instead of the old `IconMenu` the tooltips work nicely, so no need to stick to this version anymore.